langlib/lang.table/src/main/ballerina/table.bal

```diff
@@ -86,6 +114,28 @@ public isolated function get(table<MapType> key<KeyType> t, KeyType k) returns M
 # otherwise, it will be added as the last member.
 # It panics if parameter `val` is inconsistent with the inherent type of parameter `t`.
 #
+# ```ballerina
+# table<record {|int id; string name;|}> employees = table [
+#     {id: 1, name: "Jo"},
+#     {id: 2, name: "Sam"}
+# ];
+# employees.put({id: 1, name: "Pat"});
+# employees ⇒ [{"id":1,"name":"Jo"},{"id":2,"name":"Sam"},{"id":1,"name":"Pat"}]
+#
+# table<record {|readonly int id; string name;|}> key(id) students = table [
+#     {id: 1, name: "Jo"},
+#     {id: 2, name: "Sam"}
+# ];
+# 
+# students.put({id: 3, name: "Pat"});
+# students ⇒ [{"id":1,"name":"Jo"},{"id":2,"name":"Sam"},{"id":3,"name":"Pat"}]
+# students.put({id: 1, name: "Kane"});
```

Comment:
Don't we need a new line before this one?. wdyt?

Reply:
yes, It's good to have one.